.github/workflows/test.yml

```diff
@@ -0,0 +1,18 @@
+on: [push, pull_request]
+name: Test
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.17.x]
```

Comment:
Can we test with more than just the latest Go?
I don't know what version people are comfortable with. Fyne supports back to 1.14 currently so we can build for iOS 32bit devices.
I guess we should at least test against 1.16 or match what the Go team support?

Reply:
Go supports the last two minor releases at any given time (so 1.16. and 1.17 right now). I'm comfortable matching their support level, but not going all the way back to 1.14.

Reply:
I've added 1.16 to the matrix.